### ISSUE TITLE
How retrain (model) inception?

### ISSUE THREAD

**Issue by azaslavskis** [emara-geek/object-recognition-tensorflow#3]
I try retrain inception !!
When i use tensorflow 1.2.0 and 1.2.0 training model i will get mistake:
`2019-01-21 22:02:38.904542: W tensorflow/core/framework/op_def_util.cc:355] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().
2019-01-21 22:02:39.036440: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
Exception in thread "main" java.lang.IllegalArgumentException: No Operation named [softmax] in the Graph
	at org.tensorflow.Session$Runner.operationByName(Session.java:372)
	at org.tensorflow.Session$Runner.parseOutput(Session.java:381)`
When i use tensorflow 1.12.0 and 1.12.0 training model i will get mistake:
```
Exception in thread "main" java.lang.IllegalArgumentException: Invalid GraphDef
	at org.tensorflow.Graph.importGraphDef(Native Method)
	at org.tensorflow.Graph.importGraphDef(Graph.java:130)

```
I don`t now how train model ?I use  inceptionV3 
My command file is : python D://retrain.py --image_dir D:\train 
Log of train:https://nofile.io/f/QOCUJz8HW3h/log1.12.txt

**Comment by azaslavskis** [comment]
In code change

```
Tensor result = 
 s.runner().feed("DecodeJpeg/contents",image).fetch("softmax").run().get(0)) { 
```
to ` Tensor result = s.runner().feed("DecodeJpeg/contents", image).fetch("final_result").run().get(0)) {`
Download Tensorflw native libs from here : [ http://central.maven.org/maven2/org/tensorflow/libtensorflow_jni/1.6.0/libtensorflow_jni-1.6.0.jar](Here)
Third step is unpack .jar like .zip and copy lib file in your project folder.
Just get retrain.py from : [https://github.com/tensorflow/tensorflow/blob/r1.6/tensorflow/examples/image_retraining/retrain.py  ](here)
Open command prompt and enter command :  python (path to retrain.py)  --image_dir (path to image) -- output_graph (path to graph files(.pb) -- output_labels(path to label(.txt)
Find .pb file rename it to label.pb and use in code
Enjoy coding 😊
